docs/content/guides/dagster-pipes/aws-lambda.mdx

```diff
@@ -216,46 +216,16 @@ Next, you’ll add the asset and AWS Lambda resource to your project’s code lo
 
 Copy and paste the following to the bottom of `dagster_lambda_pipes.py`:
 
-```python
-# dagster_lambda_pipes.py
-
-defs = Definitions(
-  assets=[lambda_pipes_asset],
-  resources={
-    "lambda_pipes_client": PipesLambdaClient(),
-  },
-)
-```
-
-At this point, `dagster_lambda_pipes.py` should look like the following:
```

Comment:
i also removed this because they step 2.1 and 2.2 are pretty straightforward that may not need this recap (this page looks like a lot but actually not. so just trying to trimming it down a little bit)